packages/mui-material-next/src/LinearProgress/LinearProgress.types.ts

```diff
@@ -18,9 +18,15 @@ export interface LinearProgressOwnProps {
    * @default 'primary'
    */
   color?: OverridableStringUnion<
-    'primary' | 'secondary' | 'error' | 'info' | 'success' | 'warning' | 'inherit',
```

Comment:
To keep breaking changes at a minimum, we should not remove these colors, even though they're not included in the MD3 specs. This includes the `'inherit'` one.